### PR TITLE
Jjdzr4 mw 64 double password input

### PR DESCRIPTION
**Pull request by lyingparachute**
Added verification for repeated password. Password in registration is saved properly. 
Added more bootstrap to all views and deleted unnecessary comments.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,12 +12,12 @@
...
-* docker run -p 3306:3306 --name mysql -e MYSQL_ROOT_PASSWORD=password -e MYSQL_DATABASE=example --rm -d mysql
+* `docker run -p 3306:3306 --name mysql -e MYSQL_ROOT_PASSWORD=password -e MYSQL_DATABASE=example --rm -d mysql`
 ## Korzystanie z docker-compose
-* docker-compose up --build
-* docker-compose down
-* docker-compose start
-* docker-compose stop
\ No newline at end of file
+* `docker-compose up --build`
+* `docker-compose down1`
```
Comment (miosz) [inline README.md:21]: down1?

File: myFit-webApp/src/main/java/com/infoshare/myfitwebapp/controller/LoginController.java [hunk 2]
```diff
@@ -48,7 +48,11 @@ public String register(Model model) {
     }
 
     @PostMapping("register")
-    public String addUser(@Valid @ModelAttribute("user") UserLogin userLogin, Errors errors) {
+    public String addUser(Model model, @Valid @ModelAttribute("user") UserLogin userLogin, Errors errors) {
+        if (!userLogin.getPassword().equals(userLogin.getMatchingPassword())){
+            model.addAttribute("passwordIncorrect", "Passwords must match !");
```
Comment (miosz) [inline myFit-webApp/src/main/java/com/infoshare/myfitwebapp/controller/LoginController.java:53]: space character before "!" :P

File: myFit-webApp/src/main/java/com/infoshare/myfitwebapp/entity/UserLogin.java [hunk 3]
```diff
@@ -14,6 +16,7 @@
 @ToString
 @NoArgsConstructor
 @Table(name = UserLogin.TABLE_NAME)
+//@PasswordMatches
```
Comment (miosz) [inline myFit-webApp/src/main/java/com/infoshare/myfitwebapp/entity/UserLogin.java:19]: comment (can be deleted)

File: myFit-webApp/src/main/resources/messages.properties [hunk 4]
```diff
@@ -1,4 +1,18 @@
-
+indexAnonym.manageDiet=Manage your diet in a whole new way
+indexAnonym.appFeatures=Easily add products and compose your own diet based on your actual metabolism rate with the world's most popular app!
+indexAnonym.getStarted=Get Started
+indexAnonym.learnMore=Learn More
+indexAnonym.opinions=Customer opinions
+indexAnonym.opinionDesc=Our customers love our app
+indexAnonym.firstCustomerComment=Thank you for putting together such a great product. We loved working with you and the whole team, and we will be recommending you to others!
```
Comment (miosz) [inline myFit-webApp/src/main/resources/messages.properties:7]: :D

**Review by miosz — APPROVED** [review]

File: myFit-webApp/src/main/resources/messages.properties [hunk 5]
```diff
@@ -1,4 +1,18 @@
...
+indexAnonym.firstCustomerName=- Zbigniew Flaga, Wroclaw
```
Comment (mateuszostrowski) [inline myFit-webApp/src/main/resources/messages.properties:8]: Who is Zbigniew? ;)

**Review by mateuszostrowski — APPROVED** [review]

**Review by MateuszLesisz — APPROVED** [review]

**Comment by lyingparachute** [comment]
I won't delete branch until homework Ex11 and Ex12 are over

**Pull request merged**